src/extensions/service-managers/van-api-keys-manager/react-component.js

```diff
@@ -0,0 +1,80 @@
+/* eslint no-console: 0 */
+import { css } from "aphrodite";
+import PropTypes from "prop-types";
+import React from "react";
+import Form from "react-formal";
+import * as yup from "yup";
+import DisplayLink from "../../../components/DisplayLink";
+import GSForm from "../../../components/forms/GSForm";
+import GSTextField from "../../../components/forms/GSTextField";
+import GSSubmitButton from "../../../components/forms/GSSubmitButton";
+
+export class OrgConfig extends React.Component {
```

Comment:
Nice to have: would be great to support this on a per-campaign level. Maybe choose between NGP_VAN_DATABASE_MODE on the campaign level at least?